content/1.7/back-office/design/image-settings/add-edit-image-setting.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Add new Image type
+weight: 1
+---
+
+# Design > Image Settings > Add new Image type
+
+## Adding or editing Image type UI
+
+![Adding or editing Image type panel](static/img/image-settings-add-edit-image-type.png)
+
+This configuration UI is about adding the new Image type into the Image type listing.
+
```

Comment:
The following error messages are missing 
![image](https://user-images.githubusercontent.com/89604350/151189784-fdf6be22-175d-4514-81e8-7075584ca3c2.png)